plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -25,6 +25,7 @@ type CloudWatch struct {
 	Token       string `toml:"token"`
 	EndpointURL string `toml:"endpoint_url"`
 
+	DimensionsExcluded    map[string]bool `toml:"dimensions_excluded"`
```

Comment:
very small improvement but using struct{} you don't need to store the bool
```suggestion
	DimensionsExcluded    map[string]struct{} `toml:"dimensions_excluded"`
```